layouts/ucf-resource-link-card.php

```diff
@@ -16,8 +16,16 @@
 if ( ! function_exists( 'ucf_resource_link_display_card_before' ) ) {
 	function ucf_resource_link_display_card_before( $content, $args ) {
 		ob_start();
+		$label = '<li class="nav-item text-uppercase pt-1 font-weight-bold">Filter By</li>';
 	?>
 		<div class="ucf-resource-list-card-wrapper">
+		<?php if( $args['nav_position'] === "top" ): ?>
```

Comment:
Some extra flare could be added here
```suggestion
		<?php if ( $args['nav_position'] === "top" ) : ?>
```